content/vault/v1.20.x/content/docs/deploy/kubernetes/index.mdx

```diff
@@ -57,26 +57,41 @@ There are several ways to try Vault with Kubernetes in different environments.
 
 ### High level comparison of integrations
```

Comment:
This section duplicates the information in the new comparisons page so we should delete it